Make.config

```diff
@@ -304,11 +313,11 @@ MAC_COMMON_BUILD_SOURCE=$(MAC_SOURCE)/build/common
 
 ifneq ($(BUILD_REVISION),)
 # wrench build only
-IOS_MCS_FLAGS="-pathmap:$(abspath $(MONO_PATH))/=$(IOS_FRAMEWORK_DIR)/Versions/$(IOS_PACKAGE_VERSION)/src/Xamarin.iOS/" 
-IOS_MCS_FLAGS_XI="-pathmap:$(abspath $(IOS_COMMON_BUILD_SOURCE))/=$(IOS_FRAMEWORK_DIR)/Versions/$(IOS_PACKAGE_VERSION)/src/Xamarin.iOS/ -pathmap:$(abspath $(IOS_BUILD_SOURCE))/=$(IOS_FRAMEWORK_DIR)/Versions/$(IOS_PACKAGE_VERSION)/src/Xamarin.iOS/ -pathmap:$(abspath $(IOS_SOURCE))/=$(IOS_FRAMEWORK_DIR)/Versions/$(IOS_PACKAGE_VERSION)/src/Xamarin.iOS/"
+IOS_MCS_FLAGS=-pathmap:$(abspath $(MONO_PATH))/=$(IOS_FRAMEWORK_DIR)/Versions/$(IOS_PACKAGE_VERSION)/src/Xamarin.iOS/ 
+IOS_MCS_FLAGS_XI=-pathmap:$(abspath $(IOS_COMMON_BUILD_SOURCE))/=$(IOS_FRAMEWORK_DIR)/Versions/$(IOS_PACKAGE_VERSION)/src/Xamarin.iOS/ -pathmap:$(abspath $(IOS_BUILD_SOURCE))/=$(IOS_FRAMEWORK_DIR)/Versions/$(IOS_PACKAGE_VERSION)/src/Xamarin.iOS/ -pathmap:$(abspath $(IOS_SOURCE))/=$(IOS_FRAMEWORK_DIR)/Versions/$(IOS_PACKAGE_VERSION)/src/Xamarin.iOS/
```

Comment:
Why the removal of the "". Would spaces in either MONO_PATH, IOS_FRAMEWORK_DIR and others screw up the command?

Reply:
So I thought at one point it was needed but apparently not after retesting. Fixing.